assets/src/components/detourListPage.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react"
+import { DetoursTable } from "./detoursTable"
+
+export interface Detour {
+  number: number
+  direction: string
+  name: string
+  intersection: string
+  activeSince: number
+}
+
+export const DetourListPage = () => {
+  const fakeData = [
+    {
+      number: 45,
+      direction: "Outbound",
+      name: "Franklin Park via Ruggles Station",
+      intersection: "John F. Kennedy St & Memorial Drive",
+      activeSince: 1722372950,
+    },
+    {
+      number: 83,
+      direction: "Inbound",
+      name: "Central Square",
+      intersection: "Pearl Street & Clearwell Ave",
+      activeSince: 1722361948,
+    },
+    {
+      number: 83,
+      direction: "Outbound",
+      name: "Rindge Ave",
+      intersection: "Pearl Street & Clearwell Ave",
+      activeSince: 1721361948,
+    },
+    {
+      number: 45,
+      direction: "Outbound",
+      name: "Franklin Park via Ruggles Station",
+      intersection: "John F. Kennedy St & Memorial Drive",
+      activeSince: 1722372950,
+    },
+    {
+      number: 83,
+      direction: "Inbound",
+      name: "Central Square",
+      intersection: "Pearl Street & Clearwell Ave",
+      activeSince: 1722361948,
+    },
+    {
+      number: 83,
+      direction: "Outbound",
+      name: "Rindge Ave",
+      intersection: "Pearl Street & Clearwell Ave",
+      activeSince: 1721361948,
+    },
+    {
+      number: 45,
+      direction: "Outbound",
+      name: "Franklin Park via Ruggles Station",
+      intersection: "John F. Kennedy St & Memorial Drive",
+      activeSince: 1722372950,
+    },
+    {
+      number: 83,
+      direction: "Inbound",
+      name: "Central Square",
+      intersection: "Pearl Street & Clearwell Ave",
+      activeSince: 1722361948,
+    },
+    {
+      number: 83,
+      direction: "Outbound",
+      name: "Rindge Ave",
+      intersection: "Pearl Street & Clearwell Ave",
+      activeSince: 1721361948,
+    },
+  ]
+
+  return (
+    <>
+      <DetoursTable className="active" data={fakeData} />
+      <DetoursTable className="past" data={fakeData} />
```

Comment:
```suggestion
      <DetoursTable detourStatus="active" data={fakeData} />
      <DetoursTable detourStatus="past" data={fakeData} />
```

I like this abstraction, but calling it `className` confused me, since `className` typically translates directly to CSS classes, rather than being processed (by in this case, being appended to a thing).

Calling it `detourStatus` (or something else - I'm not attached to that name in particular) would make it clearer what's actually happening here.

Reply:
I think `className` is more accurate, since it serves no purpose beyond being part of the table's className. (And FWIW I have seen it done this way, though that definitely doesn't mean it's the proper way!)

Reply:
At the implementation level, it is accurate that the only effect (for now) is to update the CSS, which means that `classNameSuffix` would be an accurate name.

But from the caller's perspective, it still doesn't feel like the thing I care about. When I'm constructor a `<DetoursTable />`, I don't really care about the CSS classes applied to it; I care about whether the table is full of active detours, or past detours (or eventually draft detours), so I'd rather just explicitly specify that.

That's part of my reasoning. The other part is that when I saw `className=`, I looked in the CSS for classes called `active` or `past` and couldn't find them.

Reply:
I'll remove the prop for now! Finessing what we care about with these props feels too early for this PR, so I'll leave it for the table implementor to do.